packages/desktop-client/src/components/autocomplete/CategorySelect.tsx

```diff
@@ -71,38 +71,36 @@ function CategoryList({
           {
             overflow: 'auto',
             padding: '5px 0',
+            color: colors.altMenuItemText,
+            ...styles.altMenuText,
           },
-          !embedded && { maxHeight: 175 },
+          !embedded && { maxHeight: styles.altMenuMaxHeight },
         ]}
       >
         {items.map((item, idx) => {
           if (item.id === 'split') {
             return (
+              // Split transaction menu item
               <View
                 key="split"
                 {...(getItemProps ? getItemProps({ item }) : null)}
                 style={{
-                  backgroundColor:
-                    highlightedIndex === idx ? colors.n4 : 'transparent',
-                  borderRadius: embedded ? 4 : 0,
                   flexShrink: 0,
-                  flexDirection: 'row',
-                  alignItems: 'center',
-                  fontSize: 11,
-                  fontWeight: 500,
-                  color: colors.g8,
-                  padding: '6px 8px',
+                  margin: 5,
                 }}
                 data-testid="split-transaction-button"
               >
-                <Text style={{ lineHeight: 0 }}>
+                <Button altMenu>
```

Comment:
I did think about doing this but it is a bit different than the Payee autocomplete in that pressing the button completely changes the UI, so I think it should stay at the top. However, I agree that it should somehow be removed from the selectable list when using arrow keys.

Reply:
The “Manage Payees” button opens an entirely different page already so I don’t think that would be a huge issue. I think the best options here would be to either move it to the bottom of the list like the other buttons, or revert it to the green menu-item-like styling on `master`.

Reply:
While I can lay out the button no problem, the callback functionality is integrated very deeply and I'm not sure how to attach the right callback to the button. I don't even know how the current button gets fired. I guess since it is inside of the menu as an item, clicking the button actually fires the item callback which goes up through TransactionTable.

Reply:
The “split” item gets injected here:

https://github.com/biohzrddd/actual/blob/a6242073dfb0cdad743475d2f13d487a5460b35c/packages/desktop-client/src/components/autocomplete/CategorySelect.tsx#L178

…is then rendered where my comment is rooted, and then actually performs the action here:

https://github.com/biohzrddd/actual/blob/a6242073dfb0cdad743475d2f13d487a5460b35c/packages/desktop-client/src/components/transactions/TransactionsTable.js#L1017-L1021

You could probably pass the `onSplit` action down separately, then use the `footer` component the payee autocomplete uses to render the button in the footer when applicable.

Reply:
I could use some help on this. If you like, I can commit a moved but non-functioning button and you could hook it up?

Reply:
Sure!

Reply:
Pushed, broken and ready for a fix :)

Reply:
Pushed a fix. Tested & works locally.